client/packages/common/src/plugins/types.ts

```diff
@@ -15,9 +18,13 @@ export type Plugins = {
     StateLoader: React.ComponentType<{ stockLines: StockLineRowFragment[] }>[];
     columns: ColumnDefinition<StockLineRowFragment>[];
   };
+  requestRequisitionColumn?: {
+    StateLoader: React.ComponentType<{ requestLines: RequestLineFragment[] }>[];
+    columns: ColumnDefinition<RequestLineFragment>[];
+  };
 };
 
-type PluginData<D> = { relatedRecordId: string; data: D };
+type PluginData<D> = { relatedRecordId?: string | null; data: D };
```

Comment:
Not sure how this was passing compilation previously (in plugins), didn't want to spend too much time investigating old hooks, the new hook i added to AggrageAmcColumn example plugin did complain (since relatedRecordId was changed to be optional)